rasa/shared/utils/validation.py

```diff
@@ -168,12 +168,12 @@ def validate_yaml_schema(yaml_file_content: Text, schema_path: Text) -> None:
 
     try:
         c.validate(raise_exception=True)
-    except SchemaError:
+    except SchemaError as e:
         raise YamlValidationException(
             "Please make sure the file is correct and all "
             "mandatory parameters are specified. Here are the errors "
             "found during validation",
-            c.errors,
+            c.errors + [e],
```

Comment:
I am not sure, but can't it lead to potential duplications?

Reply:
I could add `e` if e not in `c.errors`?